c2corg_ui/static/js/map/map.js

```diff
@@ -59,7 +59,14 @@ app.module.directive('appMap', app.mapDirective);
  * @export
  * @ngInject
  */
-app.MapController = function($scope, mapFeatureCollection) {
+app.MapController = function($scope, mapFeatureCollection, $attrs) {
+
+
+  /**
+   * @type {number}
+   * @private
+   */
+  this.zoom_ = $attrs['zoom'];
```

Comment:
Maybe you need it for showing the map in the 404 page?

But do we really need to show a map in the 404 page?
I would prefer that we have no map in the 404 page (use for what?) rather than adding stuff in the JS code, useful only for it(?).